### PR TITLE
ignore case of tag and fixed bug for columName

### PR DESCRIPTION
**Pull request by ysqi**

### CODE REVIEW

File: orm/utils.go [hunk 1]
```diff
@@ -181,18 +181,36 @@ func ToInt64(value interface{}) (d int64) {
 	return
 }
 
-// snake string, XxYy to xx_yy
+// snake string, XxYy to xx_yy , XxYY to xx_yy
```
Comment (astaxie) [inline orm/utils.go]: Why change the behaves `XxYY` to `xx_yy`, I think it should be `xx_y_y`

Comment (ysqi) [inline orm/utils.go]: - ID  ---> id
- NO --->no
- OrderID ----> order_id

more see test case.

Comment (astaxie) [inline orm/utils.go]: but our rules is all the capital will change to `_char`, I admire that would make more sense. but that change the behaves maybe will broken the exist database.

Comment (ysqi) [inline orm/utils.go]: I can't think anyone use that column name `n_o` or `i_d`  . like `URL` will set tag `column(URL)`

Comment (astaxie) [inline orm/utils.go]: @ysqi could you revert this change? it looks this break a lot old implement

Comment (ysqi) [inline orm/utils.go]: @astaxie  Are You sure?

Comment (astaxie) [inline orm/utils.go]: @ysqi yes. Here is the exmaple: `GComment` need to `g_comment`, but now it turn to `gc_omment`
